### DISCUSSION TITLE
Domanda sul passaggio all'anno successivo

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#546] [Richieste di aiuto] [answered]
Buonasera,
Volevo sapere se la gestione degli scrutini per gli alunni con giudizio sospeso è possibile anche dopo il passaggio all'anno successivo.
Grazie.

**Comment by giadaschiavone** [comment]
Ciao. No, non si può, in quanto la gestione del giudizio sospeso riguarda l'anno scolastico in atto.
Dopo la gestione del giudizio sospeso e l'archiviazione dei documenti si può procedere con il nuovo anno scolastico.

**Reply by Lucaccc** [reply]
Grazie mille!

**Comment by trinko** [comment, marked as answer]
In realtà è gestito parzialmente, nel modo seguente:
- lo scrutinio del giudizio sospeso deve essere svolto ovviamente nel corrente anno scolastico, ma è possibile indicare l'esito "RINVIATO" per uno o più alunni. Questo viene motivato solitamente per problemi di salute certificati che hanno impedito lo svolgimento della verifica prevista.
- a questo punto viene prevista una sessione suppletiva per la verifica del recupero e successivamente si può attivare lo scrutinio per la **SESSIONE SUPPLETTIVA** (vedi le impostazioni in **SCUOLA > SCRUTINI**). Questi nuovi scrutini riguardano solo gli alunni con esito RIMANDATO.
- se lo scrutinio con SESSIONE SUPPLETTIVA è svolto durante l'anno scolastico, il problema è risolto.
- se, come a volte capita, è possibile svolgere la sessione supplettiva solo dopo l'inizio del nuovo anno scolstico, è attualmente necessaria un'operazione manuale per passare i dati degli alunni con esito RIMANDATO al nuovo anno scolastico;
- fatto questo, nel nuovo anno scolastico, si attiva lo scrutinio con la voce **SCRUTINI RIMANDATI** (vedi le impostazioni in **SCUOLA > SCRUTINI**) e sarà possibile svolgere lo scrutinio per gli alunni indicati, mantenendo nella documentazione i riferimenti al precedente anno scolastico.

Siccome devo effettuare una revisione della procedura di passaggio d'anno, è probabile che riesca ad automatizzare il passaggio dei dati per questi casi.

**Reply by Lucaccc** [reply]
Grazie mille!
